CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## v1.0.0 (02/12/2019)
+
+**olp-sdk-dataservice-read**
+
+* Added the `OlpClientSetting` class with the `KeyValueCache` instance.
+* Added the `VersionedLayerClient` class that is used to access versioned layers on OLP. This class implements the `getData` and `getPartitions` methods.
+* Added the `VolatileLayerClient` class that is used to access volatile layers on OLP. This class implements the `getPartitions` and `getData` methods.
+* Added the `ArtifactClient` class that is used to access the artifact service on OLP. This class implements the `getSchemaDetails` and `getSchema` methods.
+* Added the `ConfigClient` class that is used to access the configuration service on OLP. This class implements the `getCatalogs` method.
+* Added the `QueryClient` class that is used to access the query service on OLP. This class implements the `fetchQuadTreeIndex` method.
+* Added a possibility to abort requests using `AbortSignal`.
+* Added a possibility to set a billing tag in each request.
+* Added `QuadKeyUtils` that has a set of utilities and functions intended for work with quadkeys: convert a quadkey to a Morton code, convert a Morton code to a quadkey, and validate a quadkey.
+* Added the `RequestFactory` class that is used for look-up requests. This class implements the static `create` method that returns the built `DatastoreDownloadManager` instance with a correct base URL to service, and the `getBaseUrl` method that caches and returns base URLs to services.
+
+**Breaking Changes**
```

Comment:
**olp-sdk-dataservice-read** is a module while **Breaking Changes** is not. Not sure if it really denotes that the breaking changes actually belong to olp-sdk-dataservice-read module section. I would favour something more like the one from v0.9.0 and CPP SDK:

* **Breaking Change** The `CatalogClient` constructor changed. Now, pass `OlpClientSettings` by a value. Also, `KeyValueCache` is now a part of `OlpClientSettings`.

NOTE the missing `-`